### PR TITLE
balloon_check: increase the default "balloon_timeout" value

### PR DESCRIPTION
**Pull request by yanan-fu**
id:1713310

Signed-off-by: Yanan Fu <yfu@redhat.com>

### CODE REVIEW

**Comment by yanan-fu** [comment]
Test with guest memory is 63488M for 40 times, all pass.

@lijinlijin @huangyum  Please help review, thanks !

File: qemu/tests/balloon_check.py [hunk 1]
```diff
@@ -190,7 +190,7 @@ def balloon_memory(self, new_mem):
         else:
             compare_mem = new_mem
 
-        balloon_timeout = float(self.params.get("balloon_timeout", 240))
+        balloon_timeout = float(self.params.get("balloon_timeout", 360))
```
Comment (lijinlijin) [inline qemu/tests/balloon_check.py]: How about double the value to 480s, just in case fail again in high stress env.?

Comment (yanan-fu) [reply qemu/tests/balloon_check.py]: It is ok for me ~

**Comment by lijinlijin** [comment]
Acked-by: Li Jin lijin@redhat.com

**Comment by yanan-fu** [comment]
@luckyh call for your review, thanks！

**Comment by luckyh** [comment]
Thanks all.

**Pull request merged**